docker/Dockerfile_centos_deps

```diff
@@ -49,7 +49,17 @@ RUN $YUM_INSTALL centos-release-scl-rh && \
 		nasm \
 		automake \
 		which \
-		openssl-devel
+		openssl-devel \ 
+		mesa-libGL \
+		mesa-libGL-devel \
+		libXt-devel \
+		tbb-devel \
+		python36-devel \
+		libXt-devel \
+		boost-devel \
+		eigen3-devel \
```

Comment:
boost and eigen will be installed by cmake, we do not want to use OS level libraries here.